paddle/operators/momentum_op.h

```diff
@@ -50,8 +50,7 @@ class MomentumOpKernel : public framework::OpKernel<T> {
 
     v_out.device(place) = v * mu + g;
     if (use_nesterov) {
-      p_out.device(place) = p - g * lr.broadcast(grad_dsize) +
-                            v_out * mu * lr.broadcast(grad_dsize);
+      p_out.device(place) = p - (g - v_out * mu) * lr.broadcast(grad_dsize);
```

Comment:
This implement is almost same.
it's better remove the `v_out.device`

Reply:
Great, I also think it's better to remove the `v_out.device`

Reply:
I find that `v_out` and `v` point to the same memory, this is to say the writer use inplace method in python interface. And it is necessary to update `v_out` in momentum_op, so `v_out.device` can't be removed.